location/models.py

```diff
@@ -474,11 +473,14 @@ def get_user_districts(cls, user):
         """
         if hasattr(user, '_u'):
             user = user._u
-        cachedata = cache.get(f"user_districts_{user.id}")
+        cachestringkey = f"user_districts_{user.id}"
+        if LocationConfig.no_location_check :
+            cachestringkey = "user_districts_all"
+        cachedata = cache.get(cachestringkey)
         districts = []
         if cachedata is None:
             cachedata = []
-            if user.is_superuser:
+            if user.is_superuser or LocationConfig.no_location_check:
```

Comment:
we might create a specific cache for admin and when no location check to avoid filling the cache with the same data